views/template/nav.ejs

```diff
@@ -2,8 +2,18 @@
     $(function () {
         const urlParams = new URLSearchParams(window.location.search);
         const status = urlParams.get('status');
-        if(status == 's'){
+        // notification message each word in the msg should be separated by _
```

Comment:
I am getting this error in chrome when accessing `/project?status=s` make sure to fix this
`Uncaught TypeError: Cannot read property 'replaceAll' of null
    at HTMLDocument.<anonymous> (project?status=s:77)
    at j (jquery-3.2.1.min.js:2)
    at k (jquery-3.2.1.min.js:2)`